### PR TITLE
Simplify Set<String> initialization

### PR DESCRIPTION
**Pull request by balamurugana**

### CODE REVIEW

File: api/src/main/java/io/minio/MinioClient.java [hunk 1]
```diff
@@ -204,40 +204,8 @@
...
-  private static final Set<String> TRACE_QUERY_PARAMS = new HashSet<>();
-
-  static {
-    TRACE_QUERY_PARAMS.add("retention");
-    TRACE_QUERY_PARAMS.add("legal-hold");
-    TRACE_QUERY_PARAMS.add("tagging");
-    TRACE_QUERY_PARAMS.add(UPLOAD_ID);
-  }
+  private static final Set<String> TRACE_QUERY_PARAMS =
+      ImmutableSet.of("retention", "legal-hold", "tagging", UPLOAD_ID);
```
Comment (anjalshireesh) [inline api/src/main/java/io/minio/MinioClient.java:208]: I see that the variables `amzHeaders` and `standardHeaders` have been removed. Is this intentional?

Comment (balamurugana) [reply api/src/main/java/io/minio/MinioClient.java:208]: Yes. They are stale unused variables.

**Review by anjalshireesh — APPROVED** [review]
LGTM

**Review by BigUstad — APPROVED** [review]
LGTM

**Pull request merged**